src/Behat/Testwork/Tester/Exception/WrongPathsException.php

```diff
@@ -19,26 +19,41 @@
  */
 final class WrongPathsException extends RuntimeException implements TesterException
 {
+    /**
+     * @var list<string>
+     */
+    private readonly array $paths;
+
     /**
      * Initializes exception.
      *
      * @param string $message
-     * @param string $path
      */
     public function __construct(
         $message,
-        private $path,
+        string|array $paths,
     ) {
         parent::__construct($message);
+        $this->paths = (array) $paths;
+    }
+
+    /**
+     * Returns path that caused exception.
+     *
+     * @return list<string>
+     */
+    public function getPaths(): array
+    {
+        return $this->paths;
     }
 
     /**
      * Returns path that caused exception.
      *
-     * @return string
+     * @deprecated
      */
-    public function getPath()
+    public function getPath(): string
     {
-        return $this->path;
+        return implode(', ', $this->paths);
     }
```

Comment:
We don't actually use this getter, but it is part of the public interface and kept for BC. Should be very limited impact to anyone if we remove it in 4.x.